core/api/serializers/cp_report.py

```diff
@@ -58,8 +58,19 @@ class CPReportBaseSerializer(serializers.ModelSerializer):
     created_by = serializers.StringRelatedField(
         read_only=True, source="created_by.username"
     )
-    comment_country = serializers.CharField(read_only=True)
-    comment_secretariat = serializers.CharField(read_only=True)
+    comment_country_section_a = serializers.CharField(read_only=True)
```

Comment:
I tried this approach in the first place because it looked the "right" way to do it. But I noticed that I probably need 2 new models to also have per-version comments, the code became more complex and the most important thing, another call/query would be needed to get the comments for a report.